common/env.go

```diff
@@ -15,6 +16,27 @@ var (
 	commonClient *DatabricksClient
 )
 
+// GetAzureEnvironment returns the azure environment
+func GetAzureEnvironment() azure.Environment {
+	env := os.Getenv("ARM_ENVIRONMENT")
+
+	if env == "public" {
```

Comment:
I'd strongly suggest to rely on https://github.com/Azure/go-autorest/blob/master/autorest/azure/environments.go#L34 through string manipulation & https://github.com/Azure/go-autorest/blob/master/autorest/azure/environments.go#L232, because when Azure adds new cloud env - we'll have to change the code again :)

Reply:
Good point I implemented it the way you suggested